examples/case-study-1/load-db.p6

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env perl6
+
+use Red;
+
+
+use lib <./lib>;
+use OurFuncs;
+
+=begin comment
+# do not know how to 'use' these yet:
+#model Person {...}
+use Present;
+use Attend;
+use Email;
+use Person;
+=end comment
+
+#==========================================================
+# TODO Successfully split the following models into a lib
+#      directory as individual packages and 'use' them
+#      in this script and the 'query-db.p6' script.
+model Person {...}
+
+model Email is rw {
+    # relationship
+    has UInt $!person-id is referencing{:column<key>, :model<Person>};
+    has Person $.person  is relationship{:column<key>, :model<Person>};
+
+    # data:
+    has Str  $.email     is column;
```

Comment:
shouldn't it be an id?

Reply:
changed to id

Reply:
Wouldn’t be better if $.email was an I’d?

Reply:
Wouldn’t be better if $.email was a id?